lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
```

Comment:
Did we consider leveraging `jsdom` for this?

Reply:
@kratiahuja in discussions with @rwjblue we decided not to add another parser, rather use the same parser embers template system already uses: https://github.com/tildeio/simple-html-tokenizer. Future work, may want to explore using simple-html-tokenizer more often, that way we avoid having many different DOM parsing implementations around. Fully dropping JSDom would require more effort, so we should consider it case/case.

Reply:
Aah ok, thanks for the context.